hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -283,6 +294,21 @@ public AzureBlobFileSystemStore(
         "abfs-bounded");
   }
 
+  public void validateConfiguredServiceType(TracingContext tracingContext)
```

Comment:
Can this be a method inside abfsConfiguration as its about validating what's configured? GetIsNameSpaceEnabled can be the parameter passed to it.

Reply:
Makes sense. Taken